src/vonage/client.py

```diff
@@ -146,6 +149,12 @@ def api_host(self, value=None):
         else:
             self._api_host = value
 
+    def video_host(self, value=None):
+        if value is None:
+            return self._video_host
+        else:
+            self._video_host = value
```

Comment:
FYI: else is not needed since you return early above

Reply:
@manchuck the thinking for this is that it allows a user to set the `self.video_host` attribute to a new value once it's been set already. This method acts as a getter and a setter, which isn't ideal but is consistent with the existing behaviour for other pre-set endpoints.